code-challenges.test.js

```diff
@@ -15,16 +22,61 @@
 // Reminder: The test will call your function
 
 // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
+//step 1 create shell of test
+//step 2 wirte in the conditon of the test
+//step 3 creat the function 
+//step 4 run the test
+
+// describe ("divisable function",()=>{
+//     it("takes a number and divides it by 3 and returns answer", ()=>{
+//         expect(divisable(num1)).toEqual("15 is divisible by three")
+//     })// I strugled seting up the test.  I need to be able to better read why the test failed
```

Comment:
You did a great job with this whole section! Remember we can put multiple expects statements together so you don't have to copy and paste if you don't want too.